tools/ctu-ci/run_all.sh

```diff
@@ -24,7 +24,7 @@ cd projects
 rm statistics.txt
 rm detailed_stats.txt
 
-export PATH=/mnt/storage/xtu-service/clang_build/buildrelwdeb/bin:/mnt/storage/xtu-service/clang_build/codechecker/build/CodeChecker/bin:$PATH
+export PATH=/mnt/storage/ctu-service/clang_build/buildrelwdeb/bin:/mnt/storage/ctu-service/clang_build/codechecker/build/CodeChecker/bin:$PATH
```

Comment:
These paths are specific to the environment we use for testing. They should not be rewritten.